src/test/java/com/answerdigital/answerking/builder/CategoryTestBuilder.java

```diff
@@ -0,0 +1,77 @@
+package com.answerdigital.answerking.builder;
+
+import com.answerdigital.answerking.model.Category;
+import com.answerdigital.answerking.model.Product;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.answerdigital.answerking.util.DateTimeUtility.getDateTimeAsString;
+
+public class CategoryTestBuilder {
+
+    private Long id = 1L;
+    private String name;
+    private String description;
+    private String createdOn;
+    private String lastUpdated;
+    private boolean retired;
+    private Set<Product> products;
+
+    public CategoryTestBuilder withDefaultValues() {
+        this.id = 1L;
+        this.name = "Burgers";
+        this.description = "A selection of delicious burgers.";
+        this.createdOn = getDateTimeAsString();
+        this.lastUpdated = getDateTimeAsString();
```

Comment:
Subsequently, if we did switch to LocalDateTime we'd probably used LocalDateTime.now()

Reply:
Additionally, we'd have to reflect it throughout the tests - I would still keep the date utility that you created though as it seems like it could very well come in handy for other use-cases.

Reply:
Agreed. As mentioned above, the LocalDateTime refactor ticket will involve an update to the DateTimeUtility class.